getting-started/bare-metal/about.md

```diff
@@ -1,56 +1,64 @@
+When {{site.prodname}} networking is enabled, {{site.prodname}} provides the virtual networking for the pods in your cluster, allowing them to communicate with one another. It also sets up networking so that hosts can communicate with pods and vice versa. However, {{site.prodname}} does not handle networking for host to host communications; {{site.prodname}} assumes this is already set up. 
 
 When {{site.prodname}} networking is enabled...
 
 | Communication type   | Handled by         |
 |----------------------|--------------------|
-| Workload ↔ Workload  | {{site.prodname}}  |
-| Workload ↔ Host      | {{site.prodname}}  |
-| Host ↔ Host          | Underlying network |
+| pod ↔ pod            | {{site.prodname}}  |
+| pod ↔ host           | {{site.prodname}}  |
+| host ↔ host          | Underlying network |
+
+### Before you begin
+
+**Supported**
+
+- All platforms in this release except Windows
 
-This means that if you are using {{site.prodname}} networking in your cluster, and you want the non-cluster hosts to be able to communicate with workloads in the cluster, you will need to install {{site.prodname}} for networking on your non-cluster hosts.  If you are not using {{site.prodname}} for networking in your cluster (e.g. are using {{site.prodname}} in policy-only mode), or don't need your non-cluster hosts to communicate directly with workloads, you can install {{site.prodname}} in policy-only mode on your non-cluster hosts.
+**Required**
 
-#### {{site.prodname}} policy on non-cluster hosts
+- Review your install options
 
-{{site.prodname}} policy allows you to control firewalls on your non-cluster hosts using the same controls powerful controls as in your cluster.  {{site.prodname}} must be running on each non-cluster host you want to control.
+| {{site.prodname}} options     | Requires                 | Use case                                                     | Supported install methods                      |
+| ----------------------------- | ------------------------ | ------------------------------------------------------------ | ---------------------------------------------- |
+| Networking and network policy | Calico CNI               | **Networking**<br />You want {{site.prodname}} networking to handle pod-to-host, and host-to-pod communications.<br /><br />**Policy**<br />You want to use {{site.prodname}} network policy to control firewalls on your non-cluster hosts. | Docker container                               |
+| Policy only                   | A supported provider CNI | You are using a supported provider CNI for networking (for example, EKS with AWS CNI), but want to use {{site.prodname}} network policy. | Binary install with/ without a package manager |
```

Comment:
I'm not sure how CNI is relevant to non-cluster hosts - CNI is only used for pod networking, which won't exist on a non-cluster host.

Reply:
Yep, I missed some key details. Rewrote. Thanks for pointing out.